tests/test_mail/test_base.py

```diff
@@ -186,7 +189,9 @@ def test_outbox_put():
     )
     outbox.put(envelope)
     wait_for_condition(
-        lambda: inbox.empty(), 15, "Inbox must not be empty after putting an envelope"
+        lambda: inbox.empty(),
+        TIMEOUT,
+        "Inbox must not be empty after putting an envelope",
```

Comment:
Isn't this a similar case?

Reply:
lol I didn't notice. It's good that I did the refactor to introduce the `TIMEOUT` constant so you would notice this. Credits to you though for catching this!